### PR TITLE
CHE-1350: Fix empty message after successfull push with JGit

### PR DESCRIPTION
**Pull request by vinokurig**
Also chenged "Everything up-to-date" message like in Native git
@skabashnyuk please review

### CODE REVIEW

File: wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/PushTest.java [hunk 1]
```diff
@@ -71,16 +73,17 @@ public void testSimplePush(GitConnectionFactory connectionFactory)
         remoteConnection.add(newDto(AddRequest.class).withFilepattern(Arrays.asList(".")));
         remoteConnection.commit(newDto(CommitRequest.class).withMessage("Fake commit"));
         //when
-        remoteConnection.push(newDto(PushRequest.class)
-                                      .withRefSpec(Arrays.asList("refs/heads/master:refs/heads/test"))
-                                      .withRemote("origin")
-                                      .withTimeout(-1));
+        PushResponse pushResponse = remoteConnection.push(newDto(PushRequest.class)
+                                                                  .withRefSpec(Arrays.asList("refs/heads/master:refs/heads/test"))
```
Comment (skabashnyuk) [inline wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/PushTest.java]: ImmutableList.of

**Comment by skabashnyuk** [comment]
other ok
@sleshchenko ?

File: wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/PushTest.java (hunk 1 above)
Comment (vinokurig) [inline wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/PushTest.java]: done

**Comment by sleshchenko** [comment]
OK for me

**Pull request merged**